src/assets/svg/speaker.svg

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
We should be careful with svg ids, no need to change it right now but if we start noticing issues with them keep it in mind 👍

Reply:
Changed already. Thank you UNFCCC's Voice for teaching us to have always SVGs with unique nested ids.